services/espnet-tts/segment.request.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
This file should not be here.

Reply:
After I shifted my code to use absolute paths, when I execute "docker build -t tts ." in the "/services/espnet-tts"directory 
The following error raised:
Step 10/25 : COPY /services/espnet-tts/requirements.txt .
COPY failed: file not found in build context or excluded by .dockerignore: stat services/espnet-tts/requirements.txt: file does not exis